staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/controller.go

```diff
@@ -353,15 +355,30 @@ func (c *celAdmissionController) Validate(
 			}
 
 			for _, decision := range decisions {
+				paramStr := ""
+				if paramKind != nil {
+					gv, err := schema.ParseGroupVersion(paramKind.APIVersion)
+					if err == nil {
+						// only provide paramStr if we can successfully parse the groupversion
+						gvk := gv.WithKind(paramKind.Kind)
+						paramStr = gvk.String()
+					}
+				}
 				switch decision.kind {
 				case admit:
-					// TODO: add metrics for ignored error here
+					if decision.message != "" {
```

Comment:
`kind==admit && message != ""` meaning "ignored failure" is *really* hard to understand, fwiw

Reply:
I've replaced it with a `denyIgnored` field on `policyDecision`, which should be much clearer (the logic is now simply `if decision.denyIgnored`).